pom.xml

```diff
@@ -327,7 +327,7 @@
             </dependency>
             <dependency>
                 <groupId>io.netty</groupId>
-                <artifactId>netty</artifactId>
+                <artifactId>netty-all</artifactId>
```

Comment:
4.1.60 is out, since yesterday: https://search.maven.org/artifact/io.netty/netty-all/4.1.60.Final/jar

Reply:
updated